USCSandbox/UltraShaderConverter/UShader/DirectX/DirectXProgramToUSIL.cs

```diff
@@ -1288,8 +1316,8 @@ private void HandleDp2(SHDRInstruction inst)
 
             int[] mask = new int[] { 0, 1 };
             FillUSILOperand(dest, usilDest, dest.swizzle, false);
-            FillUSILOperand(src0, usilSrc0, MapMask(mask, src0.swizzle), false);
-            FillUSILOperand(src1, usilSrc1, MapMask(mask, src1.swizzle), false);
+            FillUSILOperand(src0, usilSrc0, src0.swizzle.Length >= 2 ? src0.swizzle[..2] : src0.swizzle, false);
```

Comment:
I don't see how this is any different than what MapMask does. Also, is there any case where the swizzle would be under 2 for a dp2? I would rather use MapMask for consistency.

Reply:
Tested, found it only did one good change to the decomp of one shader so this must have been done for precaution, but it doesn't appear this was needed so I reverted it